src/equations/compressible_euler_2d.jl

```diff
@@ -575,6 +575,34 @@ function boundary_condition_sedov_self_gravity(u_inner, orientation, direction,
 end
 
 
+"""
+    boundary_state_free_slip_wall(u_internal, orientation, equations::CompressibleEulerEquations2D)
```

Comment:
```suggestion
    boundary_state_free_slip_wall(u_internal, normal, equations::CompressibleEulerEquations2D)
```

Reply:
This is related to one of the issues checkboxes in PR #542 with regards to the `normal` naming convention. I copied the notation of `orientation` from other routines. It irks me to call this variable a normal here because it is not actually a normal vector. It is the normal direction that is then internally normalized. This is related to the comment above, it is super weird to me to have something like `normal = normal / norm(normal)`. It looks incredibly sloppy to me

Reply:
What do you think, @sloede?

Reply:
I am taking the first strategy and calling this variable `normal_direction` to be consistent within the other parts of the unstructured quad solver. However, this naming is something we should nail down along with the BFG guys